packages/seal-algolia-adapter/src/AlgoliaSchemaManager.php

```diff
@@ -80,6 +80,10 @@ public function dropIndex(Index $index, array $options = []): TaskInterface|null
 
     public function createIndex(Index $index, array $options = []): TaskInterface|null
     {
+        if (\count($index->distinctFields) > 1) {
+            throw new \LogicException('Algolia does not support more than one distinct field.');
+        }
```

Comment:
Created an issue to reference this in code so people can may give feedback there:

```suggestion
        if (\count($index->distinctFields) > 1) {
            throw new \LogicException('Algolia does not support more than one distinct field. See https://github.com/PHP-CMSIG/search/issues/557');
        }
```